tests/test_interfaces.py

```diff
@@ -228,3 +228,41 @@ def test_if_type_portchannel(self):
         self.assertEqual(str(value0.name), str(ObjectIdentifier(11, 0, 1, 0, (1, 3, 6, 1, 2, 1, 2, 2, 1, 3, 1003))))
         self.assertEqual(value0.data, 161)
 
+    def test_mgmt_iface(self):
+        """
+        Test that mgmt port is present in the MIB
+        """
+        oid = ObjectIdentifier(11, 0, 0, 0, (1, 3, 6, 1, 2, 1, 2, 2, 1, 1, 10000))
+        get_pdu = GetNextPDU(
+            header=PDUHeader(1, PduTypes.GET, 16, 0, 42, 0, 0, 0),
+            oids=[oid]
+        )
+
+        encoded = get_pdu.encode()
+        response = get_pdu.make_response(self.lut)
+        print(response)
+
+        value0 = response.values[0]
+        self.assertEqual(value0.type_, ValueType.INTEGER)
+        self.assertEqual(str(value0.name), str(ObjectIdentifier(11, 0, 1, 0, (1, 3, 6, 1, 2, 1, 2, 2, 1, 1, 10001))))
+        self.assertEqual(value0.data, 10000)
+
+    def test_mgmt_iface_oper_status(self):
```

Comment:
>test_mgmt_iface_oper_status [](start = 8, length = 27)

Please also add test cases for admin_status, alias.

Reply:
Alias is tested. Thanks!
Please also add test cases for admin_status

---
In reply to: [221335357](https://github.com/Azure/sonic-snmpagent/pull/88#discussion_r221335357) [](ancestors = 221335357)

Reply:
mgmt admin status test case added: https://github.com/Azure/sonic-snmpagent/pull/88/files#diff-89973946efa68dbe26ea3bd7a12d68c3R288